packages/es-dev-server/src/middleware/response-body-cache.ts

```diff
@@ -1,45 +1,41 @@
 /* eslint-disable no-restricted-syntax */
 import LRUCache from 'lru-cache';
 import fs from 'fs';
+import { Context, Middleware } from 'koa';
+import { FSWatcher } from 'chokidar';
 import { promisify } from 'util';
 import {
   getBodyAsString,
   getRequestFilePath,
   isGeneratedFile,
   RequestCancelledError,
   logDebug,
-} from '../utils/utils.js';
+} from '../utils/utils';
 
 const stat = promisify(fs.stat);
 
-/**
- * @typedef {object} ResponseBodyCacheMiddleware
- * @property {import('chokidar').FSWatcher} fileWatcher
- * @property {string} rootDir
- * @property {string[]} fileExtensions
- */
+interface ResponseBodyCacheMiddleware {
+  fileWatcher: FSWatcher;
+  rootDir: string;
+  fileExtensions: string[];
+}
 
-/**
- * @typedef {object} CacheEntry
- * @property {string} body
- * @property {object} headers
- * @property {string} filePath
- * @property {number} lastModified
- */
+interface CacheEntry {
+  body: string;
+  headers: Record<string, string>;
```

Comment:
If there's a ready-made union of header names, we should apply it here, even if it's `...|string`, for autocomplete

Reply:
In koa this is typed as `any` 🤷